src/components/breadcrumb/OpenCommentsButton.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { CommentActions, IComment } from 'tdp_comments';
+
+export interface IOpenCommentsButtonProps {
+  /**
+   * idType of the underlying workbench data.
+   */
+  idType: string;
+
+  /**
+   * Workbench selection.
+   */
+  selection: string[];
+  /**
+   * CommentPanel visibility, used to display the correct title on hover.
+   */
+  commentPanelVisible: boolean;
+
+  /**
+   * Show/hide CommentPanel.
+   */
+  onCommentPanelVisibilityChanged: (open: boolean) => void;
+}
+
+export function OpenCommentsButton({ idType, selection, commentPanelVisible, onCommentPanelVisibilityChanged }: IOpenCommentsButtonProps) {
+  const [commentCount, setCommentCount] = React.useState<number>(0);
+
+  React.useEffect(() => {
+    const listener = (_, comments: IComment[]) => {
+      // TODO: should we count the replies? The original button from tdp_comments shows the count of unread comments and + replies
```

Comment:
I'd say it should behave the same as the original comments button. I'm now not sure, if you need to uncomment the next line or if it is already implemented here like this.

Reply:
The behavior in the original comment button is that it shows the global count of new comments and replies since you(your username) last opened the panel, and it is not dependent on selection. After that, the count will be reset to 0

We currently show the total count of comments for my selected id. This is how Dominic described the button to me.
I think for our use case, since we want it to be dependent on selection, our approach is okay. If we remove the selection dependency and show only the count of new comments for my idType then that would provide more value.

Reply:
Let's keep the implementation that @dg-datavisyn described. If this is already implemented now, we can leave it as it is.